packages/harmony/src/storybook/components/ColorSwatch/ColorSwatch.tsx

```diff
@@ -10,9 +10,15 @@ type ColorSwatchProps = {
 
 export const ColorSwatch = ({ color, desc, name }: ColorSwatchProps) => {
   return (
-    <div className={styles.tile}>
-      <div className={styles.tileColor} style={{ background: color }} />
-      <div className={styles.tileInfo}>
+    <Flex
+      className={styles.tile}
+      direction='column'
+      gap='s'
+      border='strong'
+      borderRadius='xl'
+    >
+      <Box className={styles.tileColor} style={{ background: color }} />
+      <Flex direction='column' gap='xs' p='s'>
```

Comment:
wondering if `p='s'` and the like will be hard to read

I do worry about it getting too long if we don't contract stuff though. it's a big hit to readability when it starts wrapping. Maybe `padding='s'` but keep `pt, pb, pl, pr`?

Reply:
ill make a poll and see what people like 👍

Reply:
looks like people are leaning towards the abbreviated props for now. can revisit later if we find in practice that it is bad

Reply:
fine with me. Thanks!